src/shame.js

```diff
@@ -82,3 +82,21 @@ export const shamefullyAdjustGeo = ({layout: {geo = {}}}, {update}) => {
     update['geo.center'] = {};
   }
 };
+
+export const shamefullyAddTableColumns = (graphDiv, {traceIndexes, update}) => {
+  if (
+    update['cells.values'] &&
+    (!graphDiv.data[traceIndexes[0]].header ||
+      !graphDiv.data[traceIndexes[0]].header.valuessrc)
+  ) {
+    update['header.values'] = update['cells.valuessrc'];
+  } else if (update['header.values'] === null) {
+    update['header.values'] =
+      graphDiv.data[traceIndexes[0]].cells.valuessrc || null;
+  } else if (
+    update['cells.values'] === null &&
+    !graphDiv.data[traceIndexes[0]].header.valuessrc
```

Comment:
@VeraZab do we need to use your special `from`/`to` functions here to extract an array of data source names?

Reply:
I think it should be ok without any adjustments, as its the data selector itself that handles the `from` and `to` src conversions, everywhere else, the editor just uses its own format, its just in that component that we care about it, because that's where the fullValue is matched up to the dataSourceOptions

Reply:
ok, great, thanks!

Reply:
it's actually going to show very weird names in strembed, like `~:100:1000`, this does nothing to `prettify` names, its just 2 functions that allow for custom src formatting between the editor and another app. 

And those 2 functions are only used in the DataSelector, because that's the only component that has to interface between dataSourceOptions, which can be in a very custom format, and the react-chart-editor's way of handling srcs: array based for multi columns, and string based otherwise.

But these functions won't make the names of `srcs` prettier..